tests/test_transformers.py

```diff
@@ -281,3 +283,17 @@ def test_featureunion_returns_concatenated_df(categorical, numerical):
     assert isinstance(transform_df, pd.DataFrame)
     assert 8 == len(transform_df.columns)
     assert len(df) == len(transform_df)
+
+
+def test_DFStandardScaler_returns_correct_dataframe(numerical):
+    numerical_scaled = numerical
```

Comment:
```suggestion
    numerical_scaled = numerical.copy()
```
DataFrames kan returnere views ovenpå data istedenfor en kopi og det er ikke altid nemt at gætte hvornår den gør det - bedre at være eksplicit 😄